server/BraveAlerterConfigurator.js

```diff
@@ -155,6 +174,10 @@ class BraveAlerterConfigurator {
         returnMessage = t('thankYou', { lng: language })
         break
 
+      case CHATBOT_STATE.RESET:
+        returnMessage = null
```

Comment:
To follow the general chatbot principle of always sending a reply, maybe we just want to send the "Thank you" text message and `logError` the fact that we're in a state that we shouldn't ever be in?